commands/utils/consts.go

```diff
@@ -11,12 +11,18 @@ const (
 	errUnsupportedMultiRepo = "multi repository configuration isn't supported. only one repository configuration is allowed"
 
 	// Images
-	NoVulnerabilityBannerSource ImageSource = "noVulnerabilityBanner.png"
-	VulnerabilitiesBannerSource ImageSource = "vulnerabilitiesBanner.png"
-	criticalSeveritySource      ImageSource = "criticalSeverity.png"
-	highSeveritySource          ImageSource = "highSeverity.png"
-	mediumSeveritySource        ImageSource = "mediumSeverity.png"
-	lowSeveritySource           ImageSource = "lowSeverity.png"
+	NoVulnerabilityBannerSource         ImageSource = "noVulnerabilityBanner.png"
+	VulnerabilitiesBannerSource         ImageSource = "vulnerabilitiesBanner.png"
+	criticalSeveritySource              ImageSource = "criticalSeverity.png"
+	notApplicableCriticalSeveritySource ImageSource = "notApplicableCritical.png"
+	highSeveritySource                  ImageSource = "highSeverity.png"
+	notApplicableHighSeveritySource     ImageSource = "notApplicableHigh.png"
+	mediumSeveritySource                ImageSource = "mediumSeverity.png"
+	notApplicableMediumSeveritySource   ImageSource = "notApplicableMedium.png"
+	lowSeveritySource                   ImageSource = "lowSeverity.png"
+	notApplicableLowSeveritySource      ImageSource = "notApplicableLow.png"
+	unknownSeveritySource               ImageSource = "unknownSeverity.png"
+	notApplicableUnknownSeveritySource  ImageSource = "notApplicableUnknown.png"
```

Comment:
I suggest renaming the names of png files that are currently used by Frogbot, for which the images changes. This is to overcome GitHub's caching. Without renaming the files, GitHub may serve the old images from its cache.

Reply:
done